paper/main.tex

```diff
@@ -338,38 +338,37 @@ \subsubsection*{Recovering dynamic higher-order correlations}
 for longer intervals, or changed gradually (constant, ramping, and
 event datasets), recovery performance was relatively high, and we were
 better able to recover dynamic first-order correlations than
-second-order correlations.  This is because errors in our \textit{estimation}
+second-order correlations.  This is because errors in our estimation
 procedure at lower orders necessarily propagate to higher orders
 (since lower-order correlations are used to estimate higher-order
 correlations).  Conversely, when the correlations were particularly
-\textit{un}stable (random datasets), we better recovered second-order
-correlations.  This is because noise in our \textit{data generation}
+unstable (random datasets), we better recovered second-order
+correlations.  This is because noises in our data generation
 procedure propagates from higher orders to lower orders (see
-\textit{Synthetic data: simulating dynamic high-order correlations}).
+Synthetic data: simulating dynamic high-order correlations).
 
 \begin{figure}[tp]
   \centering
   \includegraphics[width=\textwidth]{figs/higher_order_sims}
-  \caption{\textbf{Recovery of simulated first-order and second-order dynamic correlations.}  Each panel displays the average correlations
+  \caption{Recovery of simulated first-order and second-order dynamic correlations.  Each panel displays the average correlations
     between the vectorized upper triangles of the recovered
     first-order and second-order correlation matrices and the true
     (simulated) first-order and second order correlation matrices at each
     timepoint and for each synthetic dataset.  (The averages are taken across 100 different
-    randomly generated synthetic datasets of each given category, each
-    with $K = 10$ features and $T = 300$ timepoints.)
```

Comment:
keep?  i think one of the reviewers asked for these details in our prior revision, right?

Reply:
yeah I really don't know why I took them out.